website/docs/d/kms_ciphertext.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "alicloud"
+page_title: "Alicloud: alicloud_kms_ciphertext"
+sidebar_current: "docs-alicloud-datasource-kms-ciphertext"
+description: |-
+    Encrypt data with KMS.
+---
+
+# alicloud\_kms\_ciphertext
+
+Encrypt a given plaintext with KMS. The plaintext gets re-encrypted on each apply, resulting in a changed ciphertext. If a stable ciphertext is needed use the [`alicloud_kms_ciphertext`](/docs/providers/alicloud/r/kms_ciphertext.html) resource.
+
+~> **NOTE**: Using this data provider will allow you to conceal secret data within your resource definitions but does not take care of protecting that data in all Terraform logging and state output. Please take care to secure your secret data beyond just the Terraform configuration.
+
+## Example Usage
+
+```
+resource "alicloud_kms_key" "key" {
+  description             = "example key"
+  is_enabled              = true
+}
+
+data "alicloud_kms_ciphertext" "encrypted" {
+  key_id    = alicloud_kms_key.key.id
+  plaintext = "example"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `plaintext` - The plaintext to be encrypted which must be encoded in Base64.
+* `key_id` - The globally unique ID of the CMK.
+* `encryption_context` -
+  (Optional) The Encryption context. If you specify this parameter here, it is also required when you call the Decrypt API operation. For more information, see [Encryption Context](https://www.alibabacloud.com/help/doc-detail/42975.htm?spm=a2c63.p38356.b99.14.47562193BvC7Hu).
+
```

Comment:
Please remove the needless string `?spm=a2c63.p38356.b99.14.47562193BvC7Hu` in the line 36.

Reply:
fixed

Reply:
fixed